azurerm/internal/services/policy/policy_assignment_resource.go

```diff
@@ -123,10 +126,41 @@ func resourceArmPolicyAssignment() *schema.Resource {
 				Optional: true,
 				Elem:     &schema.Schema{Type: schema.TypeString},
 			},
+
+			"metadata": {
+				Type:             schema.TypeString,
+				Optional:         true,
+				Computed:         true,
```

Comment:
Since we have explicitly ignored the keys that may automatically be added by the service, maybe we could remove the computed property here to enable the users to erase this attribute?

Reply:
done